demos/JWST/S5_fit_par_template.epf

```diff
@@ -70,6 +70,25 @@ u2          0.84            'free'          0           1           U
 # osc_t0          0.3163  'fixed'         0            1              U
 # osc_t1          0.3163  'free'          0.3          0.4            U
 
+# ------------------
+# ** Star Spot parameters **
+# each spot needs rad, lat, lon. see docs for details!
+# for fleck: can only have one spot contrast (spotcon0), spotnpts is the number of temporal points to evaluate at, spotrad is relative to star
+# for starry: assign one spotcon per spot (spotcon0, spotcon1, etc), spotres is the ydeg of the star map, spotrad is in degrees
+# NOTE: fleck contrast = 1 - starry contrast
+# ------------------
+# spotstari    90       'fixed'         90        1       N
+# spotrot      10       'fixed'         10        1       N
+# spotnpts     100      'independent
```

Comment:
In your notes in `ecf.rst`, you mention that this should be ~200-500 for fleck - I recommend you stick with that for this template

Reply:
done